test/fxa-geodb.js

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const chai = require('chai');
+const ERRORS = require('../lib/errors');
+const geoDb = require('../src/fxa-geodb')();
+
+const assert = chai.assert;
+
+describe('fxa-geodb', function () {
+  'use strict';
+  var ip;
+
+  it('returns a promise when called', function () {
+    assert.isTrue(typeof geoDb('12.23.34.45').then === 'function', 'Promise not returned');
```

Comment:
Chai has an `assert.isFunction` I believe.

`Promise not returned` or `promise is returned`?